dotcom-rendering/src/layouts/CommentLayout.tsx

```diff
@@ -44,10 +41,10 @@ import { SubNav } from '../components/SubNav.importable';
 import { getSoleContributor } from '../lib/byline';
 import { canRenderAds } from '../lib/canRenderAds';
 import { getContributionsServiceUrl } from '../lib/contributions';
-import { decidePalette } from '../lib/decidePalette';
 import { decideTrail } from '../lib/decideTrail';
 import { parse } from '../lib/slot-machine-flags';
 import type { NavType } from '../model/extract-nav';
+import { palette as themePalette } from '../palette';
```

Comment:
Yeah good point. I quite like naming them both because it makes them super explicit / we don't really have a palette, we have lots of palettes but happy to change.

Reply:
Really up to you, you did a different thing in #9598 so I’d opt for consistency if nothing else

Reply:
Very true! I'll update to make it consistent